modules/ols-attaching-a-resource-object-to-your-query.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is used in the following assemblies:
+// ols-using-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-attaching-a-resource-object-to-your-query_{context}"]
+= Attaching a resource object to your query
+
+This procedure explains how to attach a resource object to provided additional context for your chat query. 
+
+.Procedure
+
```

Comment:
@thoraxe the alpha doc states "If the console is currently viewing a supported resource..." 

I think I wrote up how to do that, but I took a small leap of faith in so doing. Just wanted to call out this procedure for you to review. Thanks!